library-core/src/main/kotlin/com/pusher/platform/BaseClient.kt

```diff
@@ -204,3 +215,7 @@ class BaseClient(
 
     private fun absolutePath(path: String): String = "$baseUrl/$path".replaceMultipleSlashesInUrl()
 }
+
+internal fun Request.Builder.add(headers: Headers) = headers.forEach { (name, values) ->
```

Comment:
Is this used?

Reply:
It's used in `BaseSubscription`:
https://github.com/pusher/pusher-platform-android/pull/26/files#diff-8246ed0ce0e2bf0385b2e37bb3124865R44

Could move it there or have a specific extention file for `okhttp`

Reply:
Oh, I see. I was confused by how the syntax (and self pointer) is moving around there